modules/geo-layers/src/tile-layer/tile-2d-header.js

```diff
@@ -1,13 +1,17 @@
 import {log} from '@deck.gl/core';
+import RequestScheduler from '@loaders.gl/loader-utils';
+
+const requestScheduler = new RequestScheduler({maxRequests: 1});
```

Comment:
Assume this is being set for testing, `maxRequests` should be left unset to allow the class to provide optimal values for most browsers. If so add TODO comment that this should be removed.